File: halloween/isHalloween.js
// LANGUAGE: Javascript
// ENV: Node.js
// AUTHOR: Michael Hurley
// GITHUB: https://github.com/craftman32

var currentDate = new Date();
if ((currentDate.getMonth() + 1) === 10) {
    // the current month is October
    if (currentDate.getDate() === 31) {
        // today is Halloween!
        alert('Happy Halloween!');
    } else {
        alert('It is October, but today is not Halloween!');
    }
} else {
    alert('The current month is not October!');
}

File: halloween/isHalloween.swift
import Foundation

let date = Date()
let calendar = Calendar.current
let month = calendar.component(.month, from: date)
let day = calendar.component(.day, from: date)

if (month == 10 && day == 31){
    print("Happy Halloween!")
}
else if (month == 10){
    print("It's October, but today is not Halloween!")
}
    
else {
    print("It's not October")
}

